core/src/main/proto/wallet.proto

```diff
@@ -89,8 +89,8 @@ message Key {
     /** Unencrypted - Original bitcoin secp256k1 curve */
     ORIGINAL = 1;
 
-    /** Encrypted with Scrypt and AES - Original bitcoin secp256k1 curve */
-    ENCRYPTED_SCRYPT_AES = 2;
+    /** Encrypted - Original bitcoin secp256k1 curve */
+    ENCRYPTED = 2;
```

Comment:
I could imagine this change to go into a separate PR, because it seems like it might be a reasonable simplification even without the new EncryptionType. If you agree, feel free to open a separate PR that could be discussed and merged in advance.